Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -557,11 +669,11 @@ extension SwiftLanguageServer {
           selectionRange = range
         }
 
-        let children: [DocumentSymbol]?
+        let children: [DocumentSymbol]
         if let substructure: SKDResponseArray = value[self.keys.substructure] {
           children = documentSymbols(array: substructure)
         } else {
-          children = nil
+          children = []
```

Comment:
FYI the change to `DocumentSymbol.children` is reverted in #428. We need it to remain optional in the API because that's how it's specified and clangd makes use of it being optional.